src/register/RegistrationPage.jsx

```diff
@@ -120,15 +120,17 @@ class RegistrationPage extends React.Component {
   }
 
   shouldComponentUpdate(nextProps) {
-    if (this.props.registrationFormData !== nextProps.registrationFormData) {
-      if (nextProps.registrationFormData) {
-        // do not set focused field's value from redux store to retain entered data in focused field
-        const { focusedField } = this.state;
-        const { [focusedField]: _, ...registrationData } = nextProps.registrationFormData;
-        this.setState({
-          ...registrationData,
-        });
-      }
+    if (this.props.registrationFormData !== nextProps.registrationFormData && nextProps.registrationFormData) {
+      // do not set focused field's value from redux store to retain entered data in focused field
```

Comment:
This comment is for focusedField logic and should be adjacent to `const { focusedField } = this.state;`